index.js

```diff
@@ -1,35 +1,43 @@
 'use strict';
 
-var Ajv = require('ajv');
-var jsonSchemaTest = require('json-schema-test');
-var assert = require('assert');
+const Ajv = require('ajv');
+const jsonSchemaTest = require('json-schema-test');
+const assert = require('assert');
 
-var refs = {
+const refs = {
   'http://localhost:1234/integer.json': require('./remotes/integer.json'),
   'http://localhost:1234/subSchemas.json': require('./remotes/subSchemas.json'),
   'http://localhost:1234/folder/folderInteger.json': require('./remotes/folder/folderInteger.json'),
   'http://localhost:1234/name.json': require('./remotes/name.json')
 };
 
-runTest(4);
-runTest(6);
+const SKIP = {
+  4: ['optional/zeroTerminatedFloats'],
+  7: ['optional/content', 'optional/format']
+};
 
-function runTest(draft) {
-  var opts = {
+[4, 6, 7].forEach((draft) => {
+  let opts = {
     format: 'full',
-    formats: {'json-pointer': /^(?:\/(?:[^~\/]|~0|~1)*)*$/}
```

Comment:
I'm assuming this is removed because you now recognize the json-pointer format automatically?

Reply:
Previously json-pointer format in ajv allowed hash-fragment envoding, so it had to be replaced. Now it is changed, so no need to replace it.